src/main/frontend/fs/sync.cljs

```diff
@@ -0,0 +1,1223 @@
+(ns frontend.fs.sync
```

Comment:
This is a large ns and I didn't have enough time to fully wrap my head around it. Could be helpful to add sections or code fences to it e.g.
```
### Helpers
....

### Specs
...